### PR TITLE
added bower.json & bower install method

### PR DESCRIPTION
**Pull request by Buzut**
I love your add-on as SUI really lacks one! 

I've just added a way to easily install it through Bower as it's very convenient. I added a `bower.json` file and updated the readme to show how to install via bower.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name"        : "semantic",
```
Comment (tyleryasaka) [inline bower.json]: Can this be changed to `semantic-ui-range`?

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "description" : "Semantic empowers designers and developers by creating a shared vocabulary for UI.",
```
Comment (tyleryasaka) [inline bower.json]: Can this be `Add-on range slider for Semantic UI`?

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "homepage"    : "http://www.semantic-ui.com",
```
Comment (tyleryasaka) [inline bower.json]: `https://tyleryasaka.github.io/semantic-ui-range/`

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  "author": {
+    "name" : "Jack Lukic",
```
Comment (tyleryasaka) [inline bower.json]: `Tyler Yasaka`

File: bower.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "web"  : "http://www.jacklukic.com"
```
Comment (tyleryasaka) [inline bower.json]: `https://tyleryasaka.me`

File: bower.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "dependencies": {
+    "jquery"      : ">=1.8"
+  },
+  "main": [
+    "range.css",
+    "range.js"
+  ],
+  "keywords": [
```
Comment (tyleryasaka) [inline bower.json]: We could add `range` and `slider` to this list

**Comment by Buzut** [comment]
Wooosh, I must be tired because I had done almost all of this but pushed the wrong file 👎 
I've fixed it.

**Comment by tyleryasaka** [comment]
Ha, I know how that is

**Pull request merged**